### PR TITLE
Append extra classes to existing ones in the body and html tags

### PR DESCRIPTION
**Pull request by cibernox**
Before this change, if users added classes dynamically to the body or html tags, and those tags already had any class defined, the resulting HTML would be:

```html
<html class="default-class" class="dynamic-class">
```

which is obviously invalid. Classes are afaik the only html attribute that we want to concatenate instead of just blindly appending, so I added special treatment for the class attribute only.

### CODE REVIEW

**Comment by cibernox** [comment]
@snewcomer this should fix our bug

**Comment by snewcomer** [comment]
ref #171 Notifying @ef4 if he has any interest in this PR!

File: test/fastboot-test.js [hunk 1]
```diff
@@ -138,7 +138,22 @@ describe('FastBoot', function() {
...
       .then(r => r.html())
       .then(html => {
-        expect(html).to.match(/<body data-foo=1 +class="it-works"/);
+        expect(html).to.match(
+          /<body data-before=1 +class="no-js default-class it-works" data-after=2/
+        );
+      });
+  });
+
+  it('appends classes correctly even when there was no classes in the original html', function() {
```
[the remark this answers is not in the document]
Comment (cibernox) [reply test/fastboot-test.js]: Added another test. Good point.

**Comment by kratiahuja** [comment]
Lgtm!

**Comment by cibernox** [comment]
Great, if there is anything else I must address to merge this please let me know.

**Comment by kratiahuja** [comment]
Sorry forgot to click merge last night.

**Comment by kratiahuja** [comment]
`fastboot@3.0.3` has been published

**Pull request merged**